EGO Forum Enhancement.user.js

```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
         addMoveButton(button_group, window.location.href);
 }
 
+/**
+ * Adds "On Hold" templates to the menu and increases the size of the explain box.
+ * @param {HTMLElementEventMap} event 
+ * @returns void
+ */
+function handleOnHold(event) {
+    if(event.target.nodeName != 'DIV' || !event.target.classList.contains('overlay-container'))
+        return;
+
+    // Event may fire twice - add a mark the first time it fires, and ignore the rest
+    var mark = document.createElement('input');
+    mark.type = 'hidden';
+    event.target.append(mark);
+    if(event.target.childNodes.length > 2)
```

Comment:
Inconsistent spacing

```suggestion
    if (event.target.childNodes.length > 2)
```